### PR TITLE
PSR2: require visibility only for properties and methods

### PR DESCRIPTION
**Pull request by kubawerlos**
Not for constants.

### CODE REVIEW

**Review by Maikuolan — APPROVED** [review]
Nice! This is exactly what I'm needing for where some of my own packages are utilising PHP-CS-Fixer. :-) :+1:

**Comment by mvorisek** [comment]
Why not for class constants as well?

**Comment by kubawerlos** [comment]
Because that's what is required in [PSR2](https://www.php-fig.org/psr/psr-2/):

> Visibility MUST be declared on all properties and methods;

 and always was that way in `v2` line, `v3` has different default value and that introduced the bug.

File: src/RuleSet/Sets/PSR2Set.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ public function getRules(): array
             'single_line_after_imports' => true,
             'switch_case_semicolon_to_colon' => true,
             'switch_case_space' => true,
-            'visibility_required' => true,
+            'visibility_required' => ['elements' => ['method', 'property']],
```
Comment (keradus) [inline src/RuleSet/Sets/PSR2Set.php:56]: please, add constants to the list on PSR-12

Comment (kubawerlos) [reply src/RuleSet/Sets/PSR2Set.php:56]: It's already there: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.0.0-beta.1/src/RuleSet/Sets/PSR12Set.php#L60

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**